libs/form-builder/src/lib/components/formField.component.tsx

```diff
@@ -16,12 +16,32 @@ export interface FormFieldProps {
   onClick?: (event: any) => void;
   isValidating?: boolean;
   formId?: string;
+  shouldDisplayRequiredHint?: boolean;
 }
 
-export function FormField({ id, fieldType, dictionary, ...props }: FormFieldProps) {
+export function FormField({
+  id,
+  fieldType,
+  dictionary,
+  shouldDisplayRequiredHint,
+  validation,
+  ...props
+}: FormFieldProps) {
   const Field = dictionary[fieldType];
 
   if (!Field) return null;
 
-  return <Field data-testid={id} id={id} {...props} />;
+  if (!validation || !validation?.required?.value) {
+    shouldDisplayRequiredHint = false;
+  }
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**🎯 suggestion:** ‏and the invert condition here. Should be `false` by default and set to `true` here

Reply:
The goal here is to force the boolean to be `false` if and only if the field is optionnal. If it is required, we want to use the default value.